composer.json

```diff
@@ -52,7 +52,9 @@
         "monolog/monolog": "~1.11",
         "phpunit/phpunit": "~4.5",
         "satooshi/php-coveralls": "~0.6",
-        "symfony/phpunit-bridge": "~2.8|~3.0"
+        "symfony/phpunit-bridge": "~2.8|~3.0",
+        "doctrine/orm": "^2.4.8",
```

Comment:
I found one [reference](https://github.com/contao/core-bundle/pull/540/files#diff-03e30d6a5962b13bb2fe1ad26aa7e4b5R20) to `Doctrine\ORM` though.